cmd/server/flags.go

```diff
@@ -206,6 +206,12 @@ var flags = []cli.Flag{
 		Usage:   "status context prefix",
 		Value:   "ci/woodpecker",
 	},
+	&cli.StringFlag{
+		EnvVars: []string{"WOODPECKER_STATUS_CONTEXT_FORMAT", "WOODPECKER_GITHUB_CONTEXT_FORMAT", "WOODPECKER_GITEA_CONTEXT_FORMAT"},
```

Comment:
```suggestion
		EnvVars: []string{"WOODPECKER_STATUS_CONTEXT_FORMAT"},
```
the gitea/github env var above where just keeped, to not break too much